src/components/__tests__/__snapshots__/sidebar.test.tsx.snap

```diff
@@ -447,7 +447,7 @@ Array [
     className="emotion-0"
     style={
       Object {
-        "opacity": 0,
+        "opacity": 0.00028516048692,
```

Comment:
🤨

Reply:
Tests failed over here.
These tests seem flaky as the Animation starts and slow / fast machines might have different output.

Reply:
better wait for some time and create a snapshot.

Reply:
https://github.com/ritz078/pebble/blob/23a1c815697e098489c1ea96f0b3b1ee67d6bc8f/src/components/__tests__/toast.test.tsx#L8